Orion.Net/Startup.cs

```diff
@@ -49,6 +51,16 @@ public void ConfigureServices(IServiceCollection services)
             services.AddRazorPages();
 
             services.AddSignalR();
+
+            //For Authorization
+            services.AddAuthorization(options =>
+            {
+                options.AddPolicy("PostAPI", policy =>
```

Comment:
Can you rename "PostAPI" by "SupportIDAuthentication" ?
=> PostAPI is too generic

Reply:
I used SupportID and SupportIDVerification